platform/darwin/src/MGLPolygon.h

```diff
@@ -17,6 +17,14 @@ NS_ASSUME_NONNULL_BEGIN
 @interface MGLPolygon : MGLMultiPoint <MGLOverlay>
 
 /**
+ The first coordinate of the polygon.
```

Comment:
We essentially consider this behavior to be a bug (unimplemented functionality). Instead of documenting it as a feature, we should speak about this API in general terms https://github.com/mapbox/mapbox-gl-native/pull/6211/files#r76847571 and avoid talking about future plans in the source code.